datasafe-simple-adapter/datasafe-simple-adapter-api/src/main/java/de/adorsys/datasafe/simple/adapter/api/types/DocumentDirectoryFQN.java

```diff
@@ -5,20 +5,27 @@
 import lombok.Getter;
 import lombok.ToString;
 
-@Getter
 @ToString
 public class DocumentDirectoryFQN {
     private String value;
     public DocumentDirectoryFQN(String s) {
         if (s == null) {
```

Comment:
Maybe it makes sense to check if s is not empty too, also `s` - maybe it is better to rename to `location`